cogs5e/sheets/gsheet.py

```diff
@@ -282,8 +284,16 @@ async def load_character(self, ctx, args):
         try:
             await self.get_character()
         except (KeyError, SpreadsheetNotFound, APIError):
+            if config.GOOGLE_SERVICE_ACCOUNT is not None:
+                email = Credentials.from_service_account_info(
+                    json.loads(config.GOOGLE_SERVICE_ACCOUNT),
+                    scopes=SCOPES).signer_email
+            else:
+                email = Credentials.from_service_account_file(
+                    "avrae-google.json",
+                    scopes=SCOPES).signer_email
```

Comment:
Instead of loading the credentials again on each error, I think we can use `GoogleSheet.g_client.auth.signer_email` since this `Credentials` object is stored in the client

Reply:
Ah cool! I meant to ask about that but I was waiting till Monday haha. Tested it and it works.